apps/OboeTester/app/src/main/cpp/cpu/AudioWorkloadTest.h

```diff
@@ -247,12 +257,23 @@ class AudioWorkloadTest : oboe::AudioStreamDataCallback {
     int32_t stop() {
         std::lock_guard<std::mutex> lock(mStreamLock);
         if (mStream) {
-            oboe::Result result = mStream->stop();
+            oboe::Result result = mStream->requestStop();
             if (result != oboe::Result::OK) {
                 std::cerr << "Error stopping stream: " << oboe::convertToText(result) << std::endl;
                 return static_cast<int32_t>(result);
             }
+            oboe::StreamState next;
+            result = mStream->waitForStateChange(oboe::StreamState::Stopping, &next, kTimeoutInNanos);
+            if (result != oboe::Result::OK) {
+                std::cerr << "Error while waiting for stream to stop: " << oboe::convertToText(result) << std::endl;
```

Comment:
I have questions but forgot to ask about why use `std::cerr` instead of `LOGE`?

Reply:
Thanks! This is really helpful as LOGE is non-blocking while std::cerr is blocking